scripts/peering-prefix

```diff
@@ -26,6 +26,20 @@ check_valid_asn () {  # {{{
     fi
 }  # }}}
 
+change_client_nexthop () {
+    local mux=$1
+    local mux2dev
+    mux2dev=$(grep "$mux" "$mux2dev_db")
+    local mux_offset="$((${mux2dev: -1}<<3))"
+    local cont_offset="$(($(head -n 1 "$ctxid_db")*4+2))"
+    local filterfn="$bird_filters/export_${mux}_$prefixfn.conf"
```

Comment:
This doesn't seem compatible with `do_announce` below. I think that rewriting the nexthop should be a parameter of the `announce` command. So that experimenters can specify their announcement (with any of the options supported in the `announce` command), and optionally have the nexthop configured to be the container's IP. So if I want to announce to `amsterdam01` and send traffic into the container, I can use something like:

`peering announce -m amsterdam01 -M <prefix>`

It is also worth checking if `-M` (or whatever letter we choose for routing traffic into the container) should imply `-R` (which skips configuring the data plane locally).

Reply:
Maybe we can set `config_dataplane=0` when choose -M option.